src/ReachSets/ContinuousPost/BFFPSV18/BFFPSV18.jl

```diff
@@ -41,16 +41,10 @@ function options_BFFPSV18()
                  "containing its indices"),
 
         # discretization options
-        OptionSpec(:lazy_sih, false, domain=Bool,
+        OptionSpec(:sih_method, "concrete", domain=String,
             info="use a lazy symmetric interval hull in discretization?"),
-        OptionSpec(:lazy_expm, false, domain=Bool,
-            info="use a lazy matrix exponential all the time?"),
-        OptionSpec(:lazy_expm_discretize, false, domain=Bool,
-            info="use a lazy matrix exponential in discretization?"),
-        OptionSpec(:pade_expm, false, domain=Bool,
-            info="use the Padé approximant method (instead of Julia's " *
-                 " built-in 'exp') to compute the lazy matrix exponential " *
-                 "in discretization?"),
+        OptionSpec(:exp_method, "base", domain=String,
+            info="method to compute the matrix exponential"),
```

Comment:
Can you add a `domain_check` (see other options) here?

Reply:
For this option i would avoid making the check twice, i don't see what is the advantage of it. What do you think? And also remove the `domain_check` in `:discretization`.

Reply:
The advantage is that you get a readable feedback at construction time, rather than an error message at runtime that only tells you what you did wrong if you know the algorithm.

Reply:
Another benefit is that you can read the available options in the generated docs.

Reply:
This is the standing unresolved conversation; i've opened https://github.com/JuliaReach/Reachability.jl/issues/517 to continue there.